bundle-workflow/python/test_workflow/integ_test_suite.py

```diff
@@ -6,9 +6,9 @@ def __init__(self, name, repo):
         self.repo = repo
 
     def execute(self, cluster):
-        script = self.repo.dir.name + "/integtest.sh"
+        script = self.repo.dir + "/integtest.sh"
```

Comment:
nit, os.path.join?

Reply:
That's fixed in https://github.com/opensearch-project/opensearch-build/pull/230